src/index.js

```diff
@@ -9,6 +10,13 @@ import DES from 'crypto-js/tripledes';
 import RABBIT from 'crypto-js/rabbit';
 import RC4 from 'crypto-js/rc4';
 
+const globalStorage = {
```

Comment:
Shouldn't it be:
```
const globalStorage = root.customStorage || {setItem: () => {},
  getItem: () => {},
  removeItem: () => {},
  clear: () => {}
};
```

this will allow overriding the global object and defining your own. Thoughts?

Reply:
What would `customeStorage` be?

Reply:
This would help in defining custom methods(get, set, etc.) when localStorage is not present(in case of incognito mode and cookies disabled scenario). It will help there.

Reply:
localStorage is available in incognito.

Reply:
@softvar any final thoughts?